### DISCUSSION TITLE
What is the technicaly reason that Server Side Replication is minio to minio only?

### DISCUSSION THREAD

**Discussion by deefdragon** [minio/minio#20316] [Q&A] [answered]
I'm attempting to backup the data I have in my Minio cluster to backblaze for offsite-backup/disaster recovery. I need both the files and the metadata as the metadata is used to cross reference certain information in my database. Unfortunately, it appears that I can only backup metadata via server-side replication, which only functions minio to minio at the same version.

I'm curious at this point why this restriction exists. Is the server side replication using some non-s3 api to function?

**Comment by klauspost** [comment, marked as answer]
You cannot force modification time, version IDs, SSE-C and many other properties via S3 API calls. Therefore it is impossible to upload seamless copies of objects. That is the main reason.
